conf/test_full.config

```diff
@@ -14,7 +14,21 @@ params {
     config_profile_name        = 'Full test profile'
     config_profile_description = 'Full test dataset to check pipeline function'
 
+    skip_stats                  = false
+    calc_sim                    = true
+    calc_seq_stats              = true
+    extract_plddt               = true
+    skip_eval                   = false
+    calc_sp                     = true
+    calc_tc                     = true
+    calc_irmsd                  = true
+    calc_gaps                   = true
+    calc_tcs                    = true
+
     // Input data for full size test
     input  =  params.pipelines_testdata_base_path + 'multiplesequencealign/samplesheet/v1.0/samplesheet_full.csv'
     tools  =  params.pipelines_testdata_base_path + 'multiplesequencealign/toolsheet/v1.0/toolsheet_full.csv'
+
+    // Output directory
+    outdir = "./outdir/"
```

Comment:
outdir has to be implicitly provided since this won't in aws for instance that is why it has become a mandatory parameter, please also remove it from test.config
```suggestion
```